src/java/fr/paris/lutece/plugins/forms/export/csv/FormResponseCsvExport.java

```diff
@@ -75,33 +59,43 @@ public FormResponseCsvExport( )
     {
         _csvSeparator = MultiviewConfig.getInstance( ).getCsvSeparator( );
     }
-
-    /**
-     * Build the CSV string for column line
+    /*
+     * Sort list of FormResponse by question export order
      */
+    public List<FormResponse> sortFormResponseListByQuestionOrder(List<FormResponse> formResponseList,  List<Question> listQuestions) {
+     List <FormResponse> sortedFormResponseList = new ArrayList<>();
+        HashMap<Integer, FormResponse> mapFormResponse = new HashMap<>();
+        for (int i = 0; i < listQuestions.size(); i++) {
+            Question question = listQuestions.get(i);
+          //  if (question.isResponseExportable()) {
```

Comment:
Please avoid unnecessary comments

and respect the lutece official code format, for example : 
if ( ... ) 
{
    ....
}